eggo/dag.py

```diff
@@ -96,13 +132,13 @@ def _dnload_to_local_upload_to_s3(source, destination, compression):
 
         # 3. upload to tmp S3 location
         tmp_s3_path = os.path.join(EGGO_S3_TMP_URL, random_id())
-        upload_cmd = 'pushd {tmp_dir} && aws s3 cp ./* {s3_path} && popd'
+        upload_cmd = 'pushd {tmp_dir} && aws s3 cp . {s3_path} --recursive && popd'
```

Comment:
hmm, I'm pretty sure the latest version gets installed on the cluster.  This change is fine with me as long as the result is the same.  But haven't had a problem yet.